_inc/client/state/connection/reducer.js

```diff
@@ -200,6 +217,16 @@ export function isUnlinkingUser( state ) {
 	return !! state.jetpack.connection.requests.unlinkingUser;
 }
 
+/**
+ * Returns true if currently linking the user
+ *
+ * @param  {Object} state Global state tree
+ * @return {bool} true if currently linking a user, false otherwise
+ */
+export function isAuthorizingUserInPlace( state ) {
```

Comment:
My approach here was to be explicit about iframe-based authorization, as it is special in the way that we only know that it is happening because the iframe is actually displayed.
Actually, initially I had named it `isLinkingUser` but my issue was that it was not what was actually done behind the scenes, since this is not an explicit API request we can trigger and watch.
The only thing we can be sure about, is that the auth-iframe is displayed that's why I chose to be explicit about it.